tests/scripts/thread-cert/border_router/test_radvd_coexist.py

```diff
@@ -100,10 +100,10 @@ def test(self):
         logging.info("ROUTER  addrs: %r", router.get_addrs())
         logging.info("HOST    addrs: %r", host.get_addrs())
 
-        self.assertEqual(len(br.get_prefixes()), 1)
-        self.assertEqual(len(router.get_prefixes()), 1)
-        self.assertEqual(len(br.get_routes()), 1)
-        self.assertEqual(len(router.get_routes()), 1)
+        self.assertEqual(len(br.get_netdata_omr_prefixes()), 1)
```

Comment:
Will NAT64 impact the number of prefixes? @xuyirio

Reply:
No, it won't change the number of prefixes.